### PR TITLE
docs: :memo: adds VSCode instructions for official ESlint > v9.9.0 (#268)

### PR DESCRIPTION
**Pull request by sangrepura**
ESlint > v9.9.0 includes experimental [support for Typescript configuration files](https://eslint.org/docs/latest/use/configure/configuration-files#typescript-configuration-files) using the flag `unstable_ts_config`. 
This adds  documentation for adding the flag to  VSCode.

Fixes [#201](https://github.com/AndreaPontrandolfo/sheriff/issues/201)
Fixes [#268](https://github.com/AndreaPontrandolfo/sheriff/issues/268)

### CODE REVIEW

**Comment by AndreaPontrandolfo** [comment]
Hey @sangrepura, thanks a lot for the PR!

So, there are a few things missing:

1. mention this change accordingly in the [docs page](https://www.eslint-config-sheriff.dev/docs/typescript-support/eslint-config-ts)
2. remove [eslint-ts-patch](https://github.com/antfu/eslint-ts-patch) entirely, especially from https://github.com/AndreaPontrandolfo/sheriff/tree/master/packages/sheriff-create-config
3. update the manual setup docs accordingly
4. add the changesets

**Review by AndreaPontrandolfo — APPROVED** [review]

File: apps/docs-website/docusaurus.config.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ const config: Config = {
   baseUrl: '/',
 
   future: {
-    experimental_faster: true,
+    experimental_faster: false,
```
Comment (lishaduck) [inline apps/docs-website/docusaurus.config.ts:25]: FYI, I fixed this over in #305, @AndreaPontrandolfo.

Comment (AndreaPontrandolfo) [reply apps/docs-website/docusaurus.config.ts:25]: I tried it earlier, but it didn't work for me.

Comment (AndreaPontrandolfo) [reply apps/docs-website/docusaurus.config.ts:25]: ye i tried the override, but it didnt work.

Comment (lishaduck) [reply apps/docs-website/docusaurus.config.ts:25]: Can you try now, after refreshing the lockfile? rspack 1.1.1 should've fixed it.

**Pull request merged**